public/javascripts/big.js

```diff
@@ -1976,23 +2003,26 @@ lichess.unique = function(xs) {
         });
       $form.find('div.buttons').buttonset().disableSelection();
       $form.find('button.submit').button().disableSelection();
+      var timeInputSliderIndex = 0;
```

Comment:
this code and the sequence assumes a static UI, it's better to check by the element like `.parent().hasClass('time_choice')`